playbooks/setup.yml

```diff
@@ -14,6 +14,6 @@
     - { role: nexcess.php, php_prefix: "php71" }
     - nexcess.interworx
     - nexcess.mariadb
-    - { role: nexcess.puppet, when: "nex_env_target is undefined or nex_env_target != 'vagrant'" }
+    - { role: nexcess.puppet, puppet_agent_fire_and_forget: false, when: "nex_env_target is undefined or nex_env_target != 'vagrant'" }
```

Comment:
Please put this variable in:

https://github.com/nexcess/ansible-playbook-cloudhost/blob/master/playbooks/group_vars/cloudhost.yml

Reply:
"this variable" = puppet_agent_fire_and_forget

Reply:
@clwells this has been updated.